src/components/course/course-header/CourseHeader.jsx

```diff
@@ -138,18 +138,22 @@ const CourseHeader = () => {
             />
           </Col>
           <Col xs={12}>
-            <CourseReview />
-            {defaultProgram && (
-              <p className="font-weight-bold mt-3 mb-0">
-                <FormattedMessage
-                  id="enterprise.course.about.page.course.part.of.program"
-                  defaultMessage="This course is part of a {programType}."
-                  description="Message for when a course is part of a program"
-                  values={{
-                    programType: formatProgramType(defaultProgram.type),
-                  }}
-                />
-              </p>
+            {containsContentItems && (
```

Comment:
Ensures parity with pre-feature branch (i.e., don't show `CourseReview`, etc. if course is not included in catalog.